generators/kubernetes/templates/registry/_jhipster-registry.yml

```diff
@@ -125,13 +137,13 @@ spec:
             LAST_POD_INDEX=$((${CLUSTER_SIZE} - 1))
             REPLICAS=""
             for i in $(seq 0 $LAST_POD_INDEX); do
-              REPLICAS="${REPLICAS}http://admin:${SECURITY_USER_PASSWORD}@${STATEFULSET_NAME}-${i}.jhipster-registry.${STATEFULSET_NAMESPACE}.svc.cluster.local:8761/eureka/"
+              REPLICAS="${REPLICAS}http://admin:${JHIPSTER_REGISTRY_PASSWORD}@${STATEFULSET_NAME}-${i}.jhipster-registry.${STATEFULSET_NAMESPACE}.svc.cluster.local:8761/eureka/"
               if [ $i -lt $LAST_POD_INDEX ]; then
                 REPLICAS="${REPLICAS},"
               fi
             done
-            export EUREKA_CLIENT_SERVICE_URL_DEFAULTZONE=$REPLICAS
-            echo "EUREKA_CLIENT_SERVICE_URL_DEFAULTZONE=${REPLICAS}"
+            export EUREKA_CLIENT_SERVICEURL_DEFAULTZONE=$REPLICAS
+            echo "EUREKA_CLIENT_SERVICEURL_DEFAULTZONE=${REPLICAS}"
```

Comment:
@PierreBesson @xetys - Have created [this class](https://github.com/jhipster/jhipster-registry/pull/163) to detect configMap changes to auto refresh @Refreshed scope beans. This would refresh peer url as eurekaClient is @Refresh scoped. With this, we can remove the stateful set initialisation block and most of the manual steps can be avoided. Only manual step that is required is, whenever we scale the registry corresponding url needs to be updated in jhipster-registry.xml configMap file.